wled00/FX.cpp

```diff
@@ -4861,6 +4861,53 @@ static const char _data_FX_MODE_FLOWSTRIPE[] PROGMEM = "Flow Stripe@Hue speed,Ef
 #define XY(x,y) SEGMENT.XY(x,y)
 
 
+/////////////////////////////
+//    2D Analog Clock      //
+/////////////////////////////
+uint16_t mode_2DAnalogClock(void) {                  // By Andras Fekete (bandi13)
+  if (!strip.isMatrix || !SEGMENT.is2D()) return mode_static(); // not a 2D set-up
+
+  const int cols = SEGMENT.virtualWidth();
+  const int rows = SEGMENT.virtualHeight();
+  const int centerX = (cols-!(cols%2)) >> 1;  // use odd sized circle
```

Comment:
Not terribly a fan of omitting the last row and/or column, but I can see that the circle drawn is nicer this way

Reply:
with even sized (%2 == 0) circles there is no correct center of the circle as it falls between two pixels. in such case the hands become ugly on anything smaller than 32x32 pixel